addons/mail/static/src/core/common/composer.scss

```diff
@@ -112,7 +147,11 @@
     }
 
     &::placeholder {
-        opacity: 75%;
+        opacity: 40%;
+
+        .o-mail-Composer.o-focused & {
+            opacity: 50%;
```

Comment:
10% opacity is barely a change, I suggest we use the initial 75%

Reply:
This is relatively a 20%, so still visible of a change.
I'd not increase more than 50%, the placeholder is too visible and looks like composed text. This makes using composer more exhausting than it should.